appserver/tests/payara-samples/samples-programmatic/pom.xml

```diff
@@ -29,4 +29,26 @@
             <artifactId>junit</artifactId>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>payara-server-remote</id>
```

Comment:
Samples is only run against remote so shouldn't need to have a separate profile for this - just in default

Reply:
Isn't the default profile managed?

Reply:
Maybe in the poms but we only run samples on remote

Reply:
Yes, it would be imprudent to assume that will always be the case though

Reply:
I completely understand what you mean though 😀

Reply:
going forward it needs to be run over all profiles